### PR TITLE
Fix copyright and readme

### PR DESCRIPTION
**Pull request by Adamantios**
1. Fixes the readme and copyright scripts
2. Applies the updated copyright tool to the repo

### CODE REVIEW

File: tests/__init__.py
Comment (Adamantios) [inline tests/__init__.py:1]: `tomte format-copyright --author valory` can do this now :smile:

**Review by DavidMinarsch — APPROVED** [review]

File: tomte/tools/check_readme.py [hunk 1]
```diff
@@ -39,10 +39,14 @@ def main(service_package: str) -> None:
         if wanted_hash is None or wanted_hash not in readme:
             # we were not able to find the wanted hash
             not_found = (
-                "The ElCollectooorr service package hash was not found on the readme."
+                f"The '{service_package}' service package hash was not found on the readme."
             )
             sys.exit(not_found)
 
 
 if __name__ == "__main__":
```
Comment (DavidMinarsch) [inline tomte/tools/check_readme.py:47]: Why is this needed at all?

Comment (Adamantios) [reply tomte/tools/check_readme.py:47]: Do you mean the parameter? It is unnecessary since the service package will always match the input.
Removed on 5ed321f99f218b3c1ddfe59e97ca034281ad4530.

Comment (DavidMinarsch) [reply tomte/tools/check_readme.py:47]: I mean the ability to run it as a script. This is called by CLI entrypoint so we don't need the `if __name__ == "__main__":`

Comment (Adamantios) [reply tomte/tools/check_readme.py:47]: Removed on 28ab70eb8e71439b49e7933db733b52550a03511.

**Review by DavidMinarsch — APPROVED** [review]
